experimental/WreathMacdonaldpols/src/WreathMacdonaldpols.jl

```diff
@@ -0,0 +1,256 @@
+# I would like to credit Dario Mathiä who produced an initial version of the following code
+
+import Chevie: complex_reflection_group, charinfo, CharTable, CyclotomicNumbers, Cyc
+export wreath_macs
+
+# Tools
+
+#gives indices of a subarray
+function subarray_indices(sub,arr)
+  res = Int[]
+  for i in 1:length(sub)
+    push!(res,findfirst(x->x==sub[i],arr))
+  end
+  return res
+end
+
+# Computes the b-invariant of a partition
+function b1(lambda::Vector{Int64})
```

Comment:
```suggestion
function b1(lambda::Partition)
```
from the comment above, I would expect the type to be like this

Reply:
Acutally, I need to apply b1 to the list of the sizes of all partitions in a given multipartition. It appears in the computation of the fake degree of a multipartition (cf. Stembridge "On the eigenvalues of representations of reflection
groups and wreath products", theorem 5.3).